language-support/scala/bindings/src/main/scala/com/digitalasset/ledger/client/binding/binding/Primitive.scala

```diff
@@ -101,34 +100,11 @@ sealed abstract class Primitive {
 
     private[binding] def substEx[F[_]](fa: F[rpcvalue.Identifier]): F[TemplateId[_]]
 
-    /** Package ID, module name, and entity name, unless the underlying
-      * gRPC Identifier was built with the very old and ambiguous two-argument
-      * form.  Use [[LegacyTemplateId]] instead if you want to extract
-      * the two-argument form.
+    /** Package ID, module name, and entity name.
       */
     def unapply[Tpl](t: TemplateId[Tpl]): Option[(String, String, String)]
```

Comment:
```suggestion
    def unapply[Tpl](t: TemplateId[Tpl]): Some[(String, String, String)]
```

Reply:
fixed.

Reply:
If you don't put `Some` here too it won't be visible.